app/enterprise/0.33-x/plugins/ldap-authentication-advanced.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Plugins - LDAP Authentication Advanced
+---
+# LDAP Authentication Advanced Plugin
+
+Add LDAP Bind Authentication with username and password protection. 
+The plugin will check for valid credentials in the `Proxy-Authorization` 
+and `Authorization` header (in this order).
+
+## Configuration
+
+```
+curl -X POST http://kong:8001/routes/{route_id}/plugins \
+    --data "name=ldap-auth-advanced"  \
+    --data "config.hide_credentials=true" \
+    --data "config.ldap_host=ldap.example.com" \
+    --data "config.ldap_port=389" \
+    --data "config.ldap_password=password" \
+    --data "config.start_tls=false" \
+    --data "config.base_dn=dc=example,dc=com" \
+    --data "config.verify_ldap_host=false" \
+    --data "config.attribute=cn" \
+    --data "config.cache_ttl=60" \
+    --data "config.header_type=ldap" \
+    --data "config.bind_dn=dc=example" \
+    --data "config.consumer_by=username"
+```
+
+`route_id`: the id of the Route that this plugin configuration will target.
+
+
+#### Global plugins
+
+All plugins can be configured using the `http://kong:8001/plugins/`
+endpoint. A plugin which is not associated to any Service, Route or Consumer
+(or API, if you are using an older version of Kong) is considered "global", and
+will be run on every request. Read the <a href="/latest/admin-api/#add-plugin">
+Plugin Reference</a> and the <a href="/latest/admin-api/#precedence">Plugin 
+Precedence</a> sections for more information.
+
+
+#### Parameters
+
+Here's a list of all the parameters which can be used in this plugin's configuration:
+
+Form Parameter | Default | Description
+---------------|---------|------------
+| `name` | `ldap-auth-advanced` | The name of the plugin.
+| `route_id`<br>*optional* | If present, plugin will be applied to the specified route.
```

Comment:
This needs an extra cell put in, the description text renders in the default column. Thats all I see though!

![](https://dzwonsemrish7.cloudfront.net/items/233Y0g240F2V0j333b17/Screen%20Shot%202018-07-27%20at%2010.01.41%20AM.png)